tensorflow/training/docker/2.8/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,32 @@
+{
+    "e2fsprogs": [
+        {
+            "description": " An out-of-bounds read/write vulnerability was found in e2fsprogs 1.46.5. This issue leads to a segmentation fault and possibly arbitrary code execution via a specially crafted filesystem.",
+            "vulnerability_id": "CVE-2022-1304",
+            "name": "CVE-2022-1304",
+            "package_name": "e2fsprogs",
+            "package_details": {
+                "file_path": null,
+                "name": "e2fsprogs",
+                "package_manager": "OS",
+                "version": "1.45.5",
+                "release": "2ubuntu1"
+            },
+            "remediation": {
+                "recommendation": {
+                    "text": "None Provided"
+                }
+            },
+            "cvss_v3_score": 7.8,
+            "cvss_v30_score": 0.0,
+            "cvss_v31_score": 7.8,
+            "cvss_v2_score": 6.8,
+            "cvss_v3_severity": "HIGH",
+            "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-1304.html",
+            "source": "UBUNTU_CVE",
+            "severity": "MEDIUM",
+            "status": "ACTIVE",
+            "title": "CVE-2022-1304 - e2fsprogs"
```

Comment:
Can we add a `reason_to_ignore` key stating the reason for skipping this vulnerability?